modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -2873,8 +2868,22 @@ record = new TxRecord(tx.state(), tx.nearXidVersion(), tx.writeVersion(), nodes)
         }
     }
 
-    /** Checks if a record belongs to a persistent cache with WAL enabled. */
-    private boolean containsCacheWithEnabledWal(IgniteTxAdapter tx) {
+    /**
+     * Returns {@code true} if it is possible to skip logging of transaction record to WAL.
+     *
+     * @param tx Transaction.
+     * @return {@code true} if it is possible to skip logging of transaction record to WAL.
+     */
+    private boolean canSkipTxRecordLogging(IgniteTxAdapter tx) {
```

Comment:
I am not sure that semantics of this method's return value should have been inverted. Maybe leave it as something like `txRecordLoggingNeeded`?

Reply:
Renamed.